plugins/gpio/gpioreaderthread.cpp

```diff
@@ -110,9 +110,15 @@ void ReadThread::run()
             uchar newVal = dataRead.toUInt();
             if (newVal != gpio->m_value)
             {
-                qDebug() << "Value read: GPIO:" << gpio->m_number << "val:" <<  newVal;
-                gpio->m_value = newVal;
-                emit valueChanged(gpio->m_number, gpio->m_value);
+                gpio->m_counter += 1;
+                if (gpio->m_counter > 3) {
+                    qDebug() << "Value read: GPIO:" << gpio->m_number << "val:" <<  newVal;
+                    gpio->m_value = newVal;
+                    gpio->m_counter = 0;
```

Comment:
This means that even switching from high to low requires to pass the threshold check. Doesn't make much sense to me. Once the state change is triggered, going back to the original state should be immediate